painttiming.bs

```diff
@@ -192,7 +192,7 @@ Reporting paint timing {#sec-reporting-paint-timing}
     When asked to [=mark paint timing=] given a [=Document=] |document| as input, perform the following steps:
 
     1. Let |paintTimestamp| be the [=current high resolution time=].
-    1. If this instance of [=update the rendering=] is the [=first paint=] of |document|, then invoke the [[#report-paint-timing]] algorithm with |document|, <code>"first-paint"</code>, and |paintTimestamp|.
+    1. If this instance of [=update the rendering=] is the [=first paint=] of |document|, and the user-agent is configured to mark [=first paint=], then invoke the [[#report-paint-timing]] algorithm with |document|, <code>"first-paint"</code>, and |paintTimestamp|.
```

Comment:
Dunno is "is configured" is the right term here. Maybe "supports"?

Reply:
I borrowed it from https://html.spec.whatwg.org/multipage/browsers.html#the-rules-for-choosing-a-browsing-context-given-a-browsing-context-name. 

Either works for me... WDYT?

Reply:
Interesting. I'm cool with it if @rniwa is